src/Roslyn.Diagnostics.Analyzers/Core/DeclarePublicAPIAnalyzer.Impl.cs

```diff
@@ -162,6 +159,126 @@ private static string GetErrorMessageName(ISymbol symbol, bool isImplicitlyDecla
                 return errorMessageName;
             }
 
+            private string GetSiblingNamesToRemoveFromUnshippedText(ISymbol symbol)
+            {
+                // Don't crash the analyzer if we are unable to determine stale entries to remove in public API text.
```

Comment:
Yes, basically if this additional functionality doesn't work, it doesn't block folks from at least getting the prior functionality from the fix. They will probably still complain the remove part doesn't work, and they should have a repro for us to investigate and fix.